### PR TITLE
Added Text::reduceSlashes

### PR DESCRIPTION
**Pull request by sergeyklay**

### CODE REVIEW

**Comment by Green-Cat** [comment]
:+1:

File: tests/unit/Phalcon/Text/TextReduceSlashesTest.php [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+use \PhalconTest\Text as PhTText;
+
+class TextReduceSlashesTest extends Helper\TextBase
+{
+    /**
+     * Tests the reduce slashes function
+     *
+     * @author Serghei Iakovlev <sadhooklay@gmail.com>
+     * @since  2015-09-29
```
Comment (Green-Cat) [inline tests/unit/Phalcon/Text/TextReduceSlashesTest.php]: It's the 7th of may.

Comment (sergeyklay) [inline tests/unit/Phalcon/Text/TextReduceSlashesTest.php]: Oops! :smile_cat:

**Comment by sergeyklay** [comment]
Rebased with 2.0.x

**Comment by andresgutierrez** [comment]
Could you please rebase?

**Comment by sergeyklay** [comment]
Rebased

**Comment by sergeyklay** [comment]
@andresgutierrez Sorry, rebase remaining. Now is ok.

**Comment by andresgutierrez** [comment]
Thanks

**Pull request merged**